src/utils.js

```diff
@@ -55,3 +68,14 @@ function forEachObjectKey(obj, callback) {
         }
     }
 }
+
+function createScriptElement(url, dataAttributes = {}) {
```

Comment:
Note that this new function creates the script but does not insert it into the DOM. So no network request is triggered by this function. It's reused twice above and makes it easier to compare data attributes in `findScript()`